packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1,10 +1,17 @@
-import { BROWSER_DESCRIPTION } from './constants'
+import { BROWSER_DESCRIPTION, BrowserDescription } from './constants'
 
+type Capabilities = WebDriver.Capabilities & BrowserDescription & {
```

Comment:
same, use `WebDriver.Capabilities | WebdriverIO.MultiremoteBrowserCapabilities`

Reply:
Thanks, fixed, but apparently this has to be intersection instead of union! :)